src/docs/contributing/approach/sdk-docs/write-performance.mdx

```diff
@@ -31,15 +31,21 @@ This file is `index.mdx`. It explains how to install or set up (if appropriate)
 
 To develop content for this file:
 
-1. Determine if the SDK needs an installation step or a property set to enable tracing. If so, add the SDK to `Supported` list in the `<PlatformSection>`, then add a code sample to `performance/enable-tracing/`. If not, add the SDK to `notSupported` list, which will prevent the **Enable Tracing** heading from displaying for this SDK.
+1. If the SDK is still a work-in-progress and is exposed as an experimental or an early access feature, add the platform to the `<PlatformSection>` wrapping the appropriate alert located at the beginning of the content for the page.
 
-2. Add a code sample to `/src/includes/performance/configure-sample-rate/` that covers how to set sample rates.
+1. If the SDK is expected to be used in a high-throughput environment, add the platform to the following `<PlatformSection>` which advises testing before deploying to production.
 
-3. Determine if you want to have users verify using our custom instrumentation. If so, add the SDK to `Supported` list in the `<PlatformSection>`. If not, add the SDK to `notSupported` list, which will prevent the `<PlatformLink>` to the Custom Instrumentation page from displaying for this SDK.
+1. If the SDK includes automatic and custom instrumentation, add your platform to the next `<PlatformSection>` that wraps around text that discusses this.
 
-4. Determine if the paragraph regarding pageload and navigation applies to the SDK. If so, add the SDK to `Supported` list in the `<PlatformSection>`. If not, add the SDK to `notSupported` list, which will prevent the content from displaying for this SDK.
+1. Determine if the SDK needs an installation step or a property set to enable tracing. If so, add the SDK to `Supported` list in the `<PlatformSection>` surrounding the **Enable Tracing** heading, then add a code sample to `performance/enable-tracing/`. If not, add the SDK to `notSupported` list, which will prevent the **Enable Tracing** heading from displaying for this SDK.
 
-5. Determine the level of instrumentation support for verification and add as appropriate to the `Supported` list in the `<PlatformSection>`.
+1. Add a code sample to `/src/includes/performance/configure-sample-rate/` that covers how to set sample rates. Make sure to update `/platforms/common/configuration/`.
```

Comment:
When you say update the common file here, what do you mean? I don't think there's anything to do here except the create the code sample so it can be displayed.

Reply:
Looks like I was being a little lazy here by not explicitly listing what needed to be updated. The general idea was "hey, update whatever's related to sampling and tracing in this directory (eg `sampling.mdx`, `options.mdx`)" but I'll be a little bit more specific.